api/DOMError.json

```diff
@@ -39,27 +66,42 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/DOMError/message",
           "support": {
             "chrome": {
-              "version_added": null
+              "version_added": true
```

Comment:
I guess this one takes more work to determine due to the problem of attributes on prototypes? True works for now!